tests/mcp/test_middleware.py

```diff
@@ -0,0 +1,146 @@
+"""Tests for MCP middleware configuration."""
+
+import pytest
+from fastmcp.exceptions import ToolError
+from fastmcp.server.middleware.caching import ResponseCachingMiddleware
+from fastmcp.server.middleware.error_handling import ErrorHandlingMiddleware
+
+from robocop.mcp import mcp
+from robocop.mcp.middleware import (
+    TOOLS_CACHEABLE,
+    TOOLS_NEVER_CACHE,
+    TTL_INFO,
+    TTL_METADATA,
+    TTL_RESOURCES,
+    create_caching_middleware,
+    create_error_handling_middleware,
+)
+from robocop.mcp.tools import (
+    _get_formatter_info_impl,
+    _get_rule_info_impl,
+    _lint_file_impl,
+)
+
+
+class TestMiddlewareConfiguration:
+    """Test middleware is configured correctly."""
+
+    def test_caching_middleware_created(self):
+        """Test ResponseCachingMiddleware can be created."""
+        middleware = create_caching_middleware()
+        assert middleware is not None
+
+    def test_error_handling_middleware_created(self):
+        """Test ErrorHandlingMiddleware can be created."""
+        middleware = create_error_handling_middleware()
+        assert middleware is not None

```

Comment:
In what situation this method returns None? It looks likeit creates the instance and returns it. So unless the imolemention itself changes, it will always pass